history/cultures/jinyu_group.txt

```diff
@@ -0,0 +1,66 @@
+# Ankoan
```

Comment:
They have Guerilla Warfare without living in forests.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/128549920-36defcab-b3b6-4d25-8b25-a23570923900.png)

</details>